goal_src/jak2/engine/ai/enemy.gc

```diff
@@ -157,7 +157,8 @@
   "@returns the `trans` [[vector]] from the process's `root` (typically either a [[trsqv]] or a [[collide-shape]])"
   (let ((s4-0 (-> obj root-override2)))
     (cond
-      ((zero? arg0)
+      ;; PC PORT NOTE : added lod-set check so we don't use invalid bones
+      ((or (not (logtest? (-> obj draw status) (draw-control-status lod-set))) (zero? arg0))
```

Comment:
I'm a little afraid of this change because it potentially changes a bunch of stuff for things that aren't being drawn. For example, enemies that aren't being drawn and call this function with `arg0` set to 1 used to get `gspot-pos`, and now they will get `trans`.

Reply:
Alright, changed it so the hack is a lot less aggressive and only patches when it would actually give a bad result. I don't think it's even necessary in `enemy`.